src/main/resources/templates/screens/admin/accounts/new.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html
+    xmlns:th="http://www.thymeleaf.org"
+    lang="en"
+    th:replace="~{/layouts/admin-layout :: admin-layout-fragment(
+        ~{::title},
+        ~{::#static-resources},
+        ~{::#main-content}
+    )}"
+>
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Create seller account</title>
+    <th:block id="static-resources">
+    </th:block>
+</head>
+<body id="main-content">
+<div class="container-fluid py-2">
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb bg-white mb-0">
+            <li class="breadcrumb-item"><a class="text-primary" href="#">Quản lý tài khoản</a></li>
+            <li class="breadcrumb-item active" aria-current="page">Tạo tài khoản seller</li>
+        </ol>
+    </nav>
+    <div class="col p-0">
+        <form class="needs-validation col p-0" action="/admin/accounts/new" method="post" th:object="${sellerAccount}"
+              novalidate>
+            <div class="form-group row">
+                <h3 class="col-sm-9 font-weigh-bold">Tạo tài khoản seller</h3>
+                <div class="col-sm-3 justify-content-end d-flex">
+                    <button type="submit" class="btn btn-primary rounded">Thêm mới</button>
```

Comment:
```suggestion
                    <button type="submit" class="btn btn-primary rounded">Thêm mới</button>
``` chuyển cái dòng ni về dưới mấy cái input